src/python_bindings/py_util/bind_primitive.h

```diff
@@ -56,7 +56,7 @@ auto BindPrimitive(pybind11::module_& module, auto result_method, char const* ba
                    // util::PrimitiveCollection<...>::AsList() call, the default should work,
                    // otherwise consult pybind11's docs.
                    pybind11::return_value_policy result_rv_policy =
-                           pybind11::return_value_policy::reference_internal) {
+                           pybind11::return_value_policy::copy) {
```

Comment:
I think this should be left in case someone needs it in the future for specific algorithm, plus it doesn't seem to have any effect on the compiled code in terms of time and memory.

Reply:
I put that argument in as a crutch. Nobody will need it if all primitive objects are copyable, which they should be.

Reply:
All right, I got it.

Reply:
This argument is used in some other places, so I'll leave it as is for a while until I fix other places.